Tasks/AzureRmWebAppDeployment/task.loc.json

```diff
@@ -12,8 +12,8 @@
   "author": "Microsoft Corporation",
   "version": {
     "Major": 2,
-    "Minor": 0,
-    "Patch": 5
+    "Minor": 2,
```

Comment:
As looking above task.json , I think there is some mismatch in task.json and task.loc.json. Can you generate it again.